daemon/src/taker_cfd.rs

```diff
@@ -91,7 +91,7 @@ impl<O, M> Actor<O, M> {
         cfd_feed_actor_inbox: watch::Sender<Vec<Cfd>>,
         order_feed_actor_inbox: watch::Sender<Option<Order>>,
         update_cfd_feed_sender: watch::Sender<UpdateCfdProposals>,
-        send_to_maker: Box<dyn MessageChannel<wire::TakerToMaker> + Send>,
+        send_to_maker: Box<dyn MessageChannel<wire::TakerToMaker>>,
```

Comment:
I am happy to merge minor fixes like this so if you want to do this, please make a separate commit / PR :)

Reply:
@luckysori for some reason, this one didn't land in the master - I think only maker fixes did. I was fairly sure I rebased against master before posting the PR, and doublechecked now. 
However, at this stage I might as well hold off till after the release 😉

Reply:
I'm silly, I removed the `+ Send` from the field, but not from the argument :sweat_smile: